testutils/test_server.go

```diff
@@ -293,7 +292,16 @@ func (ts *TestServer) addChannel(createChannel func(t testing.TB, opts *ChannelO
 // it waits for the channels to close.
 func (ts *TestServer) close(ch *tchannel.Channel) {
 	ch.Close()
-	ts.waitForChannelClose(ch)
+
+	select {
+	case <-time.After(500 * time.Millisecond):
```

Comment:
* use `testutils.Timeout` -- the value should be scaled up in slwoer environments.
 * The previous code did 60 iterations, each iteration taking i * 100 microseconds. i think this means we waited 100 microseconds * (1 + 2 + 3 + ... 60). That ends up being roughly 1800 * 100 microseconds or 180ms. This is a slight bump up (probably needed because we weren't using `testutils.Timeout`. I'm OK with keeping this higher, but can we test if 200ms works OK?

Reply:
I tried this with 250ms and it was still failing, https://travis-ci.org/uber/tchannel-go/builds/436414905.

Reply:
Errrp.. looks like I missed

```
export TEST_TIMEOUT_SCALE=20
```

Sounds like 200ms should work and then the Travis scaling should let it work.